### PR TITLE
Memorystore for redis

### PR DESCRIPTION
**Pull request by pk0331**
Module for creating Memory store for redis

### CODE REVIEW

File: templates/infrastructure/environment/gcp/redis/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "argonaut-name"        = "{{.Spec.name}}"
+    "argonaut-type"        = "redis"
+    "argonaut-manager"     = "argonautdev"
+    "argonaut-environment" = "{{.Environment.Name}}"
+  }
+  vpc_network_name               = dependency.{{ .Spec.vpc_id }}.outputs.network_name
+  tier                   = "{{.Spec.tier}}"
+  memory_size_gb         = "{{.Spec.memory_size_gb}}"
+  redis_version          = "{{.Spec.redis_version}}"
+  name                   = "{{.Spec.redis_version}}"
```
Comment (suryaoruganti) [inline templates/infrastructure/environment/gcp/redis/terragrunt.hcl:26]: typo

File: templates/infrastructure/environment/gcp/redis/terragrunt.hcl [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  default_labels = {
+    "argonaut-id"        = "{{.Spec.id}}"
+    "argonaut-name"        = "{{.Spec.name}}"
+    "argonaut-type"        = "redis"
+    "argonaut-manager"     = "argonautdev"
+    "argonaut-environment" = "{{.Environment.Name}}"
+  }
+  vpc_network_name               = dependency.{{ .Spec.vpc_id }}.outputs.network_name
+  tier                   = "{{.Spec.tier}}"
+  memory_size_gb         = "{{.Spec.memory_size_gb}}"
```
Comment (suryaoruganti) [inline templates/infrastructure/environment/gcp/redis/terragrunt.hcl:24]: This is a user input. I do not see it in the `modules.yaml` file @shubhamgrg04

Comment (shubhamgrg04) [reply templates/infrastructure/environment/gcp/redis/terragrunt.hcl:24]: this is not needed

File: templates/infrastructure/environment/gcp/redis/terragrunt.hcl (hunk 1 above)
Comment (shubhamgrg04) [reply templates/infrastructure/environment/gcp/redis/terragrunt.hcl:26]: resolved

**Comment by shubhamgrg04** [comment]
Merged https://github.com/argonautdev/tf-modules/pull/87

**Pull request closed without merging**